libplatsupport/src/plat/bcm2837/serial.c

```diff
@@ -17,8 +17,8 @@
 #define MU_IO       0x40
 /* When DLAB=1, MU_IIR is a baud rate register.
  * Otherwise IRQ enable */
-#define MU_IIR      0x44
-#define MU_IER      0x48
+#define MU_IIR      0x48
+#define MU_IER      0x44
```

Comment:
Please keep the numbers in ascending order.

Reply:
done

Reply:
thank, please squash the commits also

Reply:
done